.github/workflows/rule-validate.yml

```
@@ -2,7 +2,9 @@ name: Rules PR CI
 
 on:
   push:
-    branches: [ "**" ]
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "*" ]
```

strangely, you need `**` to match branches with a `/` in them. it's a glob thing, I guess
```suggestion
    branches: [ "**" ]
```